### PR TITLE
Add pangenome generate MD and MD for variable sites workflow.

### PR DESCRIPTION
**Pull request by edawson**

### CODE REVIEW

File: pangenome-generate/pangenome-generate.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+[seqwish GitHub page]().
+
+Seqwish outputs a graph in GFA format. The next step in the workflow uses [odgi]() to convert the
+graph to a succinct, indexed format that can be queried. It then builds a simple visualization of
+the graph that shows the paths of the various genomes. This graph is accessible via the handlegraph
+API, meaning it is compatible with a number of pangenome-oriented tools.
+
+### Many hands, many languages
+The workflow group packaged the pangenome construction workflow into three common workflow languages:
+CWL, NextFlow, and WDL. All of these provide interfaces to run locally as well as in the cloud and
```
Comment (bgruening) [inline pangenome-generate/pangenome-generate.md]: Galaxy as well, see https://github.com/heuermh/bh20-workflows-bhxiv/tree/master/galaxy

File: pangenome-generate/pangenome-generate.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+### Pangenome generation using minimap2, seqwish, and odgi
+[minimap2]() is a fast read mapper which can quickly align long reads.
+The inputs to pangenome construction are assembled genomes and not reads;
+in the case of SARS-CoV2, however, the viral genome is of sufficiently short length
+(roughly 30 KB) that whole genomes can be aligned in this manner. The output of
+minimap2 is a PAF file, which describes pairwise overlaps between the input sequences.
+
+
+[seqwish]() is a variation graph inducer that can transform overlapped long reads (or genomes)
```
Comment (bgruening) [inline pangenome-generate/pangenome-generate.md]: link is missing

File: pangenome-generate/pangenome-generate.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+## A polyglot workflow for pangenome construction
+Pangenomes allow the compact representation and query of many
+genomes at once. SARS-CoV2 has been sequenced extensively
+since the first weeks of the pandemic, and many full length
+genomes have been produced. The creation of a SARS-CoV2 pangenome
+from these would provide a valuable community resource that captures
+variation across samples and facilitates further research.
+
+### Pangenome generation using minimap2, seqwish, and odgi
+[minimap2]() is a fast read mapper which can quickly align long reads.
```
Comment (bgruening) [inline pangenome-generate/pangenome-generate.md]: link is missing ...

**Review by bgruening — COMMENTED** [review]
Thanks a few links are missing.

File: pangenome-generate/pangenome-generate.md (hunk 1 above)
Comment (edawson) [reply pangenome-generate/pangenome-generate.md]: Nice! Amazing that we got it in four of the workflow languages.

**Comment by heuermh** [comment]
Thank you, @edawson @bgruening!  Let me know when you're ready to merge

**Comment by heuermh** [comment]
Sorry for the delay in merging, thanks for the pull request and review!

**Pull request merged**